test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Class to generate typical test persons
+ */
+public class TypicalPersons {
+
+    public static Person amy, bill, candy, donald, grace;
```

Comment:
Shouldn't we just copy over the code from AddressBook-Level4 in order to share as much code as possible?

In AddressBook-Level4 we have alice, benson, carl etc. but here we have amy, bill, candy etc. Seems like it will cause trouble when we want to share tests across different levels in the future.